pom.xml

```diff
@@ -15,9 +15,9 @@
     </properties>
     <dependencies>
         <dependency>
-        <groupId>org.seleniumhq.selenium</groupId>
-        <artifactId>selenium-java</artifactId>
-        <version>4.5.0</version>
+            <groupId>org.seleniumhq.selenium</groupId>
+            <artifactId>selenium-java</artifactId>
+            <version>4.5.0</version>
         </dependency>
         <!-- https://mvnrepository.com/artifact/org.junit.jupiter/junit-jupiter -->
         <dependency>
```

Comment:
Clear residual dependencies (junit-jupiter-api appears twice). 
You can also take versions as a property, which is then referenced in different dependencies; will allow to easily change dependcies using same version (e.g. junit-jupiter / api / params), etc.